frontend/public/index.html

```diff
@@ -7,6 +7,7 @@
       href="https://fonts.googleapis.com/css?family=VT323&display=swap"
       rel="stylesheet"
     />
+    <link href="/your-path-to-fontawesome/css/all.css" rel="stylesheet" />
```

Comment:
? I'll need an explanation on this one

Reply:
This is the link for using the icon from the music button

Reply:
`your-path-to-fontawesome` ?